syncthing_gtk/app.py

```diff
@@ -983,6 +983,7 @@ def get_folder_n_path(self, path):
 		relative path in that folder, or (None, None) if specified path
 		doesn't belongs anywhere
 		"""
+		path = path.decode("utf-8")
```

Comment:
> Also I don't know whether it's the right place to do the conversion.

Probably not... Can I ask where does error occurs? Does it generate stack trace?